libs/crypto/include/crypto/fnv.hpp

```diff
@@ -54,9 +55,11 @@ class FNV : public HasherInterface
   std::size_t HashSizeInBytes() const override;
 
 private:
-  internal::FnvHasherInternals *impl_;
+  std::shared_ptr<internal::FnvHasherInternals> impl_;
 };
 
+inline FNV::~FNV() = default;
```

Comment:
This can be moved to the main body of the header surely

Reply:
:+1: